### PR TITLE
Drop Ubuntu 10/12/14 code

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -130,14 +130,6 @@
           'log_mode'    => '0755',
           'run_dir'     => '/run/nginx',
         }
-      } else {
```
Comment (ekohl) [inline manifests/params.pp:133]: Why drop this? Doesn't Ubuntu 18.04 hit this? I also think it'd be a better and safer thing to have an `else` clause. If anything, Xenial should perhaps not manage the repo at all.

Comment (bastelfreak) [reply manifests/params.pp:133]: the syntax isn't perfect here, above this block is one explicitly for Debian 9/10 and Ubuntu 18

Comment (ekohl) [reply manifests/params.pp:133]: Still, having a safe default feels better to me. I'd also be in favor of dropping `manage_repo => true` and moving to not managing it by default. Then we can make the else block the future proof one so Debian 11/Ubuntu 20.04 are more likely to just work.

Comment (bastelfreak) [reply manifests/params.pp:133]: I'm not sure if we should change any default here. However I'm fine with providing a default block that catches newer ubuntu/debian version to be future-proof. My personal preference is also to always use the upstream repositories and not the packages from Ubuntu.

**Comment by ekohl** [comment]
> I feel that the `manage_repo` behavior should be documented in the changelog somehow.

Can we open an issue for this and let this PR close it? Then it's documented.

**Comment by bastelfreak** [comment]
closes https://github.com/voxpupuli/puppet-nginx/issues/1441

**Pull request merged**